sov-modules/sov-modules-macros/tests/dispatch/derive_dispatch.rs

```diff
@@ -7,6 +7,7 @@ use sov_modules_macros::{DispatchCall, DispatchQuery, Genesis, MessageCodec};
 use sov_state::ProverStorage;
 
 #[derive(Genesis, DispatchCall, DispatchQuery, MessageCodec)]
+#[serialization(serde::Serialize, serde::Deserialize)]
```

Comment:
Here it should be `borsh::BorshDeserialize/BorshSerialize`. Maybe the issue was not clear. We want to replace the hardcoded `borsh::BorshDeserialize/BorshSerialize` inside macro with macro attributes and the PR is very close to it (just replace `serde` with borsh)